### ISSUE TITLE
v15 BlockList + inline editing + nested blocks is broken

### ISSUE THREAD

**Issue by dknippers** [umbraco/Umbraco-CMS#17756]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

15.0.0

### Bug summary

BlockList with inline editing and nested blocks is broken. Nested blocks are added to the wrong block. 

This occurs in at least 15.0.0 and 15.1.0-rc but perhaps other versions. It looks very similar to #17312 which was v14.3.0 so likely this has been broken for a long time. This seems to occur with any property that opens a modal, not just nested blocks. Like in the linked issue this happens with a URL picker but this probably happens with pretty much all pickers making this completely unusable.

In my case having a BlockList with inline editing and a block type that nests itself cannot be used as nested blocks are added to the wrong block. Have a look at the video, where I cannot add a Nested block to Block 2, they will always be added to Block 1. 

Also note a bugged `%1%` label in the "Create Block for ..." button.

You can import these uSync files to reproduce it (will create doctype + element type + block list datatype)
[uSync.zip](https://github.com/user-attachments/files/18049664/uSync.zip)

![blocks](https://github.com/user-attachments/assets/2acaa0f0-9680-485b-9674-ef9bca1b2793)

### Steps to reproduce

See the video + attached uSync files to reproduce, but in short

1. Add a block with a nested block
2. Add a second block
3. Try to add a nested block to block 2
4. Observe it will be added under block 1 instead

### Expected result / actual result

Working UI

---
_This item has been added to our backlog AB#47398_

**Comment by NguyenThuyLan** [comment]
You're right, @dknippers , it seems to be the same problem as issue #17312

**Comment by leekelleher** [comment]
This issue has been fixed in #17929.
